x-pack/legacy/plugins/task_manager/README.md

```diff
@@ -305,6 +328,36 @@ server.plugins.task_manager.addMiddleware({
 });
 ```
 
+## Task Poller: polling for work
+TaskManager used to work in a `pull` model, but it now needs to support both `push` and `pull`, so it has been remodeled internally to support a single `push` model.
+
+Task Manager's lifecycle is pushed by the following operations:
```

Comment:
Well, actually, I _did_ mean pushed... language is hard, this might be my multilingual brain having trouble distinguishing which word belongs in which language.

Reworded.